--- ember-cordova/cordova/plugins/cordova-plugin-fcm/src/android/FCMPlugin.gradle.orig	2017-05-19 18:04:04.418718549 +0200
+++ ember-cordova/cordova/plugins/cordova-plugin-fcm/src/android/FCMPlugin.gradle	2017-05-19 18:05:02.097932263 +0200
@@ -5,9 +5,9 @@
         }
     dependencies {
         classpath 'com.android.tools.build:gradle:+'
-        classpath 'com.google.gms:google-services:+'
+        classpath 'com.google.gms:google-services:3.0.0'
     }
 }
 // apply plugin: 'com.google.gms.google-services'
 // class must be used instead of id(string) to be able to apply plugin from non-root gradle file
-apply plugin: com.google.gms.googleservices.GoogleServicesPlugin
\ No newline at end of file
+apply plugin: com.google.gms.googleservices.GoogleServicesPlugin
